jerry-core/debugger/jerry-debugger.h

```diff
@@ -82,12 +107,17 @@ typedef enum
   JERRY_DEBUGGER_EVAL_ERROR_END = 20, /**< last part of eval result when an error is occured */
 
   /* Messages sent by the client to server. */
+    /* The following messages are accepted in both run and breakpoint modes. */
```

Comment:
This indentation was intended from my side. Perhaps I could bring the `/*`one space earlier, and add a white space after it (so there will be two spaces after the `/*`). What do you think?